routers/repo/view.go

```diff
@@ -62,13 +62,13 @@ func renderDirectory(ctx *context.Context, treeLink string) {
 			continue
 		}
 
-		if !markup.IsReadmeFile(entry.Name()) {
-			continue
+		if markup.IsReadmeFile(entry.Name(), true) {
+			readmeFile = entry.Blob()
+			break
 		}
 
-		readmeFile = entry.Blob()
-		if markup.Type(entry.Name()) != "" {
-			break
+		if markup.IsReadmeFile(entry.Name(), false) {
```

Comment:
We need a priority of readme file here: `readme.md` > `readme.txt` > `readme` > `readme.<i18n>.md` > ... ?

Reply:
Done